05/app.js

```diff
@@ -8,6 +8,27 @@ const stats = {
 };
 
 /* tutaj umieść swój kod */
+const paragraphAll = document.querySelectorAll('[data-id]');
+const p1 = document.querySelector('[data-id=p1]');
+
+function sumClickHandler (e) {
+    e.preventDefault();
+    let clickedItem = e.currentTarget;
+    // zliczanie klików dla p1
+    if(clickedItem === p1) {
```

Comment:
Musimy sprawdzać klikanie we wszystkie paragrafy, a nie tylko pierwszy (podobnie z linkami).
Trzeba tak to przerobić, aby rozwiązanie było uniwersalne.
Jak będzie 100 paragrafów i 200 linków statystyki mają zawierać informacje o każdym kliknięciu :)